File: Semestrul 4/MPP2/ConcursAtletism/src/main/java/ro/mpp2025/domain/Enrollment.java
package ro.mpp2025.domain;

import java.util.ArrayList;
import java.util.List;

public class Enrollment implements Entity<Integer> {
    private Integer id;
    private final Child child;
    private final Event event;

    public Enrollment(Child child, Event event) {
        this.child = child;
        this.event = event;
    }

    public Child getChild() {
        return child;
    }

    public Event getEvent() {
        return event;
    }

    @Override
    public void setId(Integer id) {
        this.id = id;
    }

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public String toString() {
        return id + "|" + child + "|" + event;
    }
}

File: Semestrul 4/MPP2/Problema8_FX_Csharp/Problema8_FX_Csharp/Domain/Enrollment.cs
namespace Problema8_FX_Csharp.Domain;

public class Enrollment: Entity<long>
{
    public Child Child { get; set; }
    public Event Event { get; set; }
    
    public Enrollment(long id, Child child, Event eventt)
    {
        SetId(id);
        Child = child;
        Event = eventt;
    }
    
    public override String ToString()
    {
        return $"Child: {Child}, Event: {Event}";
    }
}